### ISSUE TITLE
Added the ability to edit custom fonts for cards!!

### ISSUE THREAD

**Issue by kogakisaki** [AKORA-Studios/DiscordWelcomeCard#77]

**Comment by DragonCat4012** [comment]
Yes you can use custom fonts

**Comment by DragonCat4012** [comment]
> can i make a welcome card on whatsapp or telegram bot? with this module?

Yes you can use it for other platforms as well. You can use the create custom card function for that. (The default welcome and goodbye card functions won't work since they need a discord member Object)
